feature/mypage/src/main/java/com/terning/feature/mypage/mypage/MyPageRoute.kt

```diff
@@ -94,8 +94,12 @@ fun MyPageRoute(
         rememberPermissionState(permission = notificationPermission)
     var isChecked by remember {
```

Comment:
이 변수는 상태 객체 자체가 바뀌지 않고, 내부 값만 변경되는 구조라서 val로 선언해도 괜찮을 것 같아요!
```suggestion
    val isChecked = remember {
```

Reply:
말씀해주신 부분 보고 저도 생각을 해 보았는데요..! 

사실 저는 `by`를 써주면 `isChecked` 변수를 사용할 때 `.value`를 같이 사용하지 않아도 돼서 가독성이 좋아진다고 생각했었어요..!
그런데 이렇게 `by`를 써주게 되면 get/set에 대한 위임이 이루어지면서 값이 바뀔 수 있는 var로 사용되어야 하더라구요
val로 선언하고 `.value`를 같이 사용해주는 것이 좋을까요? 

다른 분들의 의견도 궁금하네요!!

Reply:
var의 경우 상태 객체 자체를 바꿀 수 있어서 조금의 실수라도 방지하기 위해 val을 사용하는게 좋다고 생각했습니다!

Reply:
조금 더 찾아보니까 by를 사용하면 상태 값 자체에 접근할 수 있고 효빈이 말처럼 객체 자체를 바꾸진 못한다고 하네요! 
이 상황에선 간결성을 위해 `var`을 유지하는게 맞는 것 같아요!
덕분에 하나 배워갑니다~